app/services/mlh_table.rb

```diff
@@ -17,17 +17,17 @@ def faraday_connection
     ) do |faraday|
       faraday.use Faraday::Response::RaiseError
       faraday.adapter Faraday.default_adapter
-    end
-    response = @faraday_connection.get
-    unless Rails.configuration.cache_store == :null_store
-      response.body do
-        ActiveSupport::Cache.lookup_store(
-          *Rails.configuration.cache_store,
-          namespace: 'mlh',
-          expires_in: 3.hours
-        )
+      faraday.response :caching do
+        unless Rails.configuration.cache_store == :null_store
```

Comment:
I think the unless block needs to be outside the `:caching` block? -- with it inside in the default Docker setup, I get a nil error.

Reply:
Where are you seeing the nil error?

Reply:
If I pull this branch down locally and go to /events, I get:

![image](https://user-images.githubusercontent.com/12371363/95120037-d523dd00-0744-11eb-858d-c19f5c3c8242.png)

Reply:
Does this happen after you run the `stop-and-clean` script and reboot? I can't seem to reproduce even after wiping my local setup.

Reply:
On a completely fresh set of containers with a fresh copy of this branch, yup, this is what I am seeing. I'm assuming my cache store is the null store (no caching), so as the `unless` is inside the `caching` block, the caching for the connection is being set to `nil`, which is causing this issue.

If we move the `unless` outside the `caching` block, we'll only set caching on the connection when there is a valid cache store available.